src/main/java/frc/team4909/robot/subsystems/climber/ClimberSubsystem.java

```diff
@@ -0,0 +1,58 @@
+package frc.team4909.robot.subsystems.climber;
+
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import com.ctre.phoenix.motorcontrol.can.WPI_VictorSPX;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+import edu.wpi.first.wpilibj.SpeedControllerGroup;
+import edu.wpi.first.wpilibj.command.Subsystem;
+import edu.wpi.first.wpilibj.drive.DifferentialDrive;
+import frc.team4909.robot.RobotMap;
+import frc.team4909.robot.RobotConstants;
+import frc.team4909.robot.subsystems.drivetrain.commands.Drive;
+import frc.team4909.robot.subsystems.climber.commands.ExtendStilts;
+import frc.team4909.robot.subsystems.climber.commands.RetractStilts;
+import com.ctre.phoenix.motorcontrol.FeedbackDevice;
+import edu.wpi.first.wpilibj.command.CommandGroup;
+import frc.team4909.robot.Robot;
+import frc.team4909.robot.operator.controllers.BionicF310;
+
+public class ClimberSubsystem extends Subsystem {
+    WPI_TalonSRX climberSRX;
```

Comment:
this naming needs some work, that's indecipherable anywhere throughout the code. take the lines 25 & 26 and apply them here

ex. `climberLiftSRX` and `climberDriveSPX`